conf/production.ini

```diff
@@ -89,4 +89,9 @@ py-tracebacker = /tmp/via-traceback-
 #env = VIA_IGNORE_PREFIXES=https://hypothes.is/,https://qa.hypothes.is/,https://cdn.hypothes.is/
 
 mount = /=viahtml/wsgi.py
-uwsgi-socket = /tmp/viahtml-uwsgi.sock
\ No newline at end of file
+uwsgi-socket = /tmp/viahtml-uwsgi.sock
+
+stats =  0.0.0.0:3033
```

Comment:
I tried this with a unix file socket but:

 * It's not any quicker
 * It involves installing a different library
 * ... or doing some horrible socket stuff